static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -244,15 +243,18 @@ export const WidgetInteractiveTitle = ({
       options={menuOptions}
       value={chartSetting}
       onChange={handleChange}
-      renderWrapAs={TextOverflow}
       triggerProps={{borderless: true, size: 'zero'}}
     />
   );
 };
 
 const StyledCompactSelect = styled(CompactSelect)`
+  min-width: 0;
+  font-weight: 400;
```

Comment:
Reset font weight inside the select menu.

**Before:**
<img width="218" alt="Screenshot 2023-01-11 at 3 06 11 PM" src="https://user-images.githubusercontent.com/44172267/211937267-e9ce296e-6747-4ed8-92b5-8c61755123a8.png">


**After:**
<img width="218" alt="Screenshot 2023-01-11 at 3 06 59 PM" src="https://user-images.githubusercontent.com/44172267/211937337-f55f2d5b-62ca-4006-8041-893bc1490502.png">